### PR TITLE
rolling upgrade: align base versions in jenkins jobs

### PR DESCRIPTION
**Pull request by eliransin**
The base versions vector for the jenkins upgrade jobs included only 4.4
which is not correct. This is probably because the 2021.1 SCT version
was branched out of the master branch.
We change the base versions vector to be: 2020.1, 4.3 and 2021.1

### CODE REVIEW

File: jenkins-pipelines/rolling-upgrade-alternator.jenkinsfile
Comment (amoskong) [inline jenkins-pipelines/rolling-upgrade-alternator.jenkinsfile:8]: The base_version change is branch special.

We don't support migrate from 2020.1 to master, and 2021.1 to master, so they can't be added to base_versions of master.

If we use SCT master to test Enterprise (enterprise branch), it's fine. And SCT pipeline will filter out the unsupported base_versions.

Comment (eliransin) [reply jenkins-pipelines/rolling-upgrade-alternator.jenkinsfile:8]: Sorry, my bad, changed the branch (I wanted to merge it into branch-2021.1)

**Review by amoskong — APPROVED** [review]
Looks good.

**Comment by eliransin** [comment]
@amoskong thanks - what else should I don in order to get it merged?

**Comment by amoskong** [comment]
> @amoskong thanks - what else should I don in order to get it merged?

Wait for @scylladb/qa-maintainers to review and merge it.

**Comment by bentsi** [comment]
@eliransin
1) please correct the commit message (read the [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI))
2) Don't delete the PR prechecks template:

image

File: jenkins-pipelines/rolling-upgrade-ami.jenkinsfile
Comment (bentsi) [inline jenkins-pipelines/rolling-upgrade-ami.jenkinsfile:8]: why do you need to upgrade from '2021.1' to the same version, '2021.1'?

Comment (amoskong) [reply jenkins-pipelines/rolling-upgrade-ami.jenkinsfile:8]: The `2021.1` in base_version is official released (2021.1.x), the destination version will be unstable 2021.1.y

Comment (bentsi) [reply jenkins-pipelines/rolling-upgrade-ami.jenkinsfile:8]: got it

**Comment by eliransin** [comment]
Thanks @bentsi and @amoskong :smile:

**Pull request merged**
